tests/zenko_tests/node_tests/backbeat/IngestionUtility.js

```diff
@@ -0,0 +1,170 @@
+const assert = require('assert');
+const async = require('async');
+const crypto = require('crypto');
+
+const { scalityS3Client, ringS3Client } = require('../s3SDK');
+
+class IngestionUtility {
+    constructor(s3, ringS3C) {
+        this.s3 = s3;
+        this.ringS3C = ringS3C;
+    }
+
+    _compareObjectBody(body1, body2) {
+        const digest1 = crypto.createHash('md5').update(body1).digest('hex');
+        const digest2 = crypto.createHash('md5').update(body2).digest('hex');
+        assert.strictEqual(digest1, digest2);
+    }
+
+
+    _deleteVersionList(versionList, bucketName, cb) {
+        async.each(versionList, (versionInfo, next) =>
+            this.deleteObject(bucketName, versionInfo.Key,
```

Comment:
You can optimize this later by using S3's Batch Delete api to do this in one request.